GraphicEditor/View/Windows/EquipmentSearch.xaml.cs

```diff
@@ -10,9 +10,11 @@ namespace GraphicEditor.View.Windows
     /// </summary>
     public partial class EquipmentSearch : Window
     {
-        private EquipmentRepository equipmentRepository = new EquipmentRepository();
-        private RoomRepository roomRepository = new RoomRepository();
-        private MedicineRepository medicineRepository = new MedicineRepository();
+        private EquipmentDatabaseSql equipmentRepository = new EquipmentDatabaseSql();
+        private RoomDatabaseSql roomRepository = new RoomDatabaseSql();
+        private MedicineDatabaseSql medicineRepository = new MedicineDatabaseSql();
+        private FloorDatabaseSql floorRepository = new FloorDatabaseSql();
+        private BuildingDatabaseSql buildingRepository = new BuildingDatabaseSql();
```

Comment:
![image](https://user-images.githubusercontent.com/57258204/102021278-10fba400-3d7f-11eb-824f-0f39cd27db98.png)
You need to set up project architecture and work with that, so the view model (not view) should call the 'model' or in this case our backend code - starting with controller, controller calls services and they communicate with the database and do the business logic

Reply:
The methods in backend aren't implemented yet. At this moment we are not able to make whole CRUD for backend. We will call and implement controllers etc. in next sprint. :))